javalang/parser.py

```diff
@@ -854,6 +854,15 @@ def parse_field_declarators_rest(self):
         return tree.FieldDeclaration(declarators=declarators)
 
     @parse_debug
+    def parse_lambda_method_body(self):
```

Comment:
Could you move this next to `parse_lambda_expression` just so they're near one another?